File: 03 godisnja_doba/godisnja_doba.py
dan, mesec = [int(i) for i in input().split()]

godDoba = ""
if (mesec == 1 or mesec == 2 or (mesec == 3 and dan <= 20) or (mesec == 12 and dan > 21)):
    godDoba = "ZIMA"
elif (mesec == 4 or mesec == 5 or (mesec == 3 and dan > 21) or (mesec == 6 and dan <= 21)):
    godDoba = "PROLECE"
elif (mesec == 7 or mesec == 8 or (mesec == 6 and dan > 21) or (mesec == 9 and dan <= 22)):
    godDoba = "LETO"
else:
    godDoba = "JESEN"

print(godDoba)

File: 03 godisnja_doba/godisnja_doba.cs
using System;

namespace godisnja_doba
{
    class Program
    {
        static void Main(string[] args)
        {
            string[] ulaz = Console.ReadLine().Split();
            int dan = int.Parse(ulaz[0]);
            int mesec = int.Parse(ulaz[1]);
            string godDoba = "";
            switch (mesec)
            {
            case 1:
            case 2:
                godDoba = "ZIMA";
                break;
            case 3:
                if (dan <= 20) godDoba = "ZIMA";
                else godDoba = "PROLECE";
                break;
            case 4:
            case 5:
                godDoba = "PROLECE";
                break;
            case 6:
                if (dan <= 21) godDoba = "PROLECE";
                else godDoba = "LETO";
                break;
            case 7:
            case 8:
                godDoba = "LETO";
                break;
            case 9:
                if (dan <= 22) godDoba = "LETO";
                else godDoba = "JESEN";
                break;
            case 10:
            case 11:
                godDoba = "JESEN";
                break;
            case 12:
                if (dan <= 21) godDoba = "JESEN";
                else godDoba = "ZIMA";
                break;
            }
            Console.WriteLine(godDoba);
        }
    }
}
